pkg/kudoctl/cmd/version.go

```diff
@@ -30,5 +34,40 @@ func newVersionCmd() *cobra.Command {
 func VersionCmd(cmd *cobra.Command, args []string) error {
 	kudoVersion := version.Get()
 	fmt.Printf("KUDO Version: %s\n", fmt.Sprintf("%#v", kudoVersion))
+
+	// Print the Controller Version
+	controllerVersion, err := GetControllerVersion()
+	if err != nil {
+		fmt.Printf("KUDO Controller Version: %s\n", controllerVersion)
+	} else {
+		fmt.Printf("KUDO Controller Version: %#v\n", err)
```

Comment:
Let's return an error instead.

```suggestion
		return fmt.Errorf("failed to retrieve KUDO controller version: %v\n", err)
```